docs/docs-components/MarkdownPage.js

```diff
@@ -198,20 +198,26 @@ const components = {
     alt,
     width,
     height,
-    noPadding,
+    addPadding,
   }: {|
     src: string,
     caption?: string,
     alt?: string,
     width?: number,
     height?: number,
-    noPadding?: boolean,
+    addPadding?: boolean,
```

Comment:
While I do appreciate you flipping this prop (good practice when dealing with boolean props), let's try a slightly different approach:
```js
padding?: 'standard' | 'none',
```
where "none" is the default value? That gives us flexibility to add other padding options in the future while avoiding a boolean prop for something that isn't inherently boolean ("padding" could have any number of values)

Reply:
Makes sense! Fixed